### ISSUE TITLE
geo scanner and portible geo scanner not working

### ISSUE THREAD

**Issue by Dutch13112** [Slimefun/Slimefun4#3557]
### ❗ Checklist

- [X] I am using the official english version of Slimefun and did not modify the jar.
- [X] I am using an up to date "DEV" (not "RC") version of Slimefun.
- [X] I am aware that issues related to Slimefun addons need to be reported on their bug trackers and not here.
- [X] I searched for similar open issues and could not find an existing bug report on this.

### 📍 Description

anything i tried with slimefun i cant get the geo scanner or portible version of it to work only thing im getting is this error in the console also yes i tried reinstalling and reset my whole server nothing worked pelease help me.

p.s. for the  error look at the paste bin and i dont know how to replicate it

### 📑 Reproduction Steps

i dont know how to repoduce it

### 💡 Expected Behavior

you dont get a menu or anything when you use them

### 📜 Server Log

https://pastebin.com/wkvu40jL

### 💻 Server Software

Paper

### 🎮 Minecraft Version

1.18.x

### ⭐ Slimefun version

dev 1014 (git f431a)

**Comment by ybw0014** [comment]
Looks like you have downloaded a corrupted jar, redownload a new one.

**Comment by Dutch13112** [comment]
i found the problem it was not with the corrupted jar i had a plugin called new beginnings still dont know why it was there or what it did but that was giving the problems

**Comment by TheBusyBiscuit** [comment]
Yep, the error indicates it's caused by "NewBeginnings", not Slimefun.
